### PR TITLE
Ports a big ass comically large lollipop from /TG/Station

### PR DESCRIPTION
**Pull request by ClownMoff**
## About The Pull Request
Since in https://github.com/BeeStation/BeeStation-Hornet/pull/10640 i'm adding the propeller hat and the bowtie, i'm also adding the lolipop. Original PR is https://github.com/tgstation/tgstation/pull/80034

## Why It's Good For The Game
It's fun and dumb. I added a recipe so it can be crafted by the cook. It's usually found in hacked snack vendors. All sales finals.

<details>
<summary> Me running down the hallways with a big ass lolipop in hand</summary>

</details>

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

imagen
imagen

</details>

## Changelog
:cl: ClownMoff, san7890
add: Adds a swirl lolipop from /TG/ Station. Found in hacked snack machines and crafteable under the "Misc" section.
/:cl:

### CODE REVIEW

**Comment by Tsar-Salat** [comment]
286788306-185ad63b-8221-4050-9897-9df8cde5d1fd

File: code/modules/food_and_drinks/recipes/tablecraft/recipes_misc.dm [hunk 1]
```diff
@@ -185,3 +185,12 @@
...
 	result = /obj/item/reagent_containers/food/snacks/pingles
 	subcategory = CAT_MISCFOOD
+/datum/crafting_recipe/food/swirl_lollipop
+	name = "swirl lollipop"
+	reqs = list(
+		/datum/reagent/consumable/sugar = 5,
+		/datum/reagent/consumable/caramel = 5,
+		/datum/reagent/drug/happiness = 5,
+		)
+	result = /obj/item/food/lollipop/swirl_lollipop
```
Comment (Tsar-Salat) [inline code/modules/food_and_drinks/recipes/tablecraft/recipes_misc.dm]:
```suggestion
	result = /obj/item/food/lollipop/swirl
```

Comment (ClownMoff) [reply code/modules/food_and_drinks/recipes/tablecraft/recipes_misc.dm]: I already noticed thanks

**Comment by Rukofamicom** [comment]
image

You're becoming a real coder

**Review by Penwin0 — APPROVED** [review]

**Pull request merged**